model.py

```diff
@@ -0,0 +1,32 @@
+# Разработал: Куликовских Юлия Валентиновна
+# 'model.py' - модуль, отвечающий за модель данных и операции с ними. Он содержит функции для работы 
+# с данными контактов, включая загрузку контактов из CSV-файла, сохранение контактов в CSV-файл и 
+# поиск контактов по ключевому слову.
+
+import os
+
+import csv
+
+# Функция для загрузки контактов из CSV-файла
+def load_contacts(file_name):
+    contacts = []
+    try:
+        with open(file_name, "r", encoding="utf-8") as file:
+            reader = csv.reader(file)
+            for row in reader:
+                contacts.append(row)
+    except FileNotFoundError:
+        pass
+    return contacts
```

Comment:
Можно не обрабатывать исключение, если немного подумать
```suggestion
    open(file_name, "a", encoding="utf-8").close()
    with open(file_name, "r", encoding="utf-8") as file:
        reader = csv.reader(file)
        for row in reader:
            contacts.append(row)
    return contacts
```

Reply:
У еще можно читать проще
```suggestion
    open(file_name, "a", encoding="utf-8").close()
    with open(file_name, "r", encoding="utf-8") as file:
        for row in file:
            contacts.append(row)
    return contacts
```